Monika After Story/game/script-moods.rpy

```diff
@@ -556,3 +556,72 @@ label mas_mood_yearolder_years:
 label mas_mood_yearolder_leap_today:
     # nothing special occurs here for now
     jump mas_mood_yearolder_bday_true
+
+init 5 python:
+    addEvent(Event(persistent._mas_mood_database,"mas_mood_scared",prompt="scared",category=[store.mas_moods.TYPE_BAD],unlocked=True),eventdb=store.mas_moods.mood_db)
+
+label mas_mood_scared:
+    m 1r "{i}There is nothing to fear, but fear itself.{/i}"
+    m 1k "Ahaha~ I'm kidding."
+    m 3a "Though the reason we become scared, is because we sense any form of danger."
+    m "This can be in a physical or psychological form."
+    m 1d "Fear is a survival instinct that we are born with, and this trait keeps us from harm."
+    m 1g "Though, this instinct can also restrict our actions that may be benefitial to us, instead of damaging."
+    m 1e "I hope your reasons behind being scared, is keeping you from harm [player]."
+    m 1f "I would never want to see my beloved sweetheart hurt."
+    return
+
+init 5 python:
+    addEvent(Event(persistent._mas_mood_database,"mas_mood_guilty",prompt="guilty",category=[store.mas_moods.TYPE_BAD],unlocked=True),eventdb=store.mas_moods.mood_db)
+
+label mas_mood_guilty:
+
+    m 1f "Aw! You shouldn't feel guilty [player]."
+    m 2g "There is no way that you could know or do anything so bad to feel that way."
+    m 4p "I mean, doing what I did to my friends was something to feel guilty over."
+    m 3e "But if it will make you feel better, you can confess to me what's on your mind."
+    $ start_time = datetime.datetime.now()
+    m 3a "Whisper it or say it out loud to me. You can trust me."
+    m 1q "..."
+    $ end_time = datetime.datetime.now()
+    $ elapsed_time = end_time - start_time
+    $ time_threshold = datetime.timedelta(seconds=20)
+    if elapsed_time < time_threshold:
+        m 2f "I guess you don't have to tell me if you're too uncomfortable."
+        m "That's fine, you can always come back to tell me anytime."
+        m 2e "I hope you won't be afraid to tell me anything on your mind."
+        m 1k "You know I'll always love you [player]."
+    elif elapsed_time < time_threshold * 3:
+        m 2b "Thanks for confronting to me [player]."
```

Comment:
This line makes no sense. D:
"Thanks for coming to me, [player]."
"Thanks for confronting me [player]".

Confrontation is a negative thing, it's a hostile (not necessarily violent) action and not one somebody does when comforting them. I'd re-do this line.